### ISSUE TITLE
The auto import not working on storybook

### ISSUE THREAD

**Issue by alvinzjoe** [lazercaveman/nuxt-starter#15]
Hi @lazercaveman, 

I've found issue on storybook the nuxt 3 auto-import component not working. 

Thanks!

**Comment by lazercaveman** [comment]
@alvinzjoe  Yes, you are correct. Nuxt auto-imports do not work in Storybook by default because Storybook operates independently of Nuxt. While Storybook can display Nuxt components, it does not have access to the same configuration and features that Nuxt provides, such as auto-imports.

Maybe the  [@nuxtjs/storybook](https://nuxt.com/modules/storybook) module could help integrate Nuxt with Storybook - so using the nuxt-storybook module may be a better option, as it provides tighter integration between Nuxt and Storybook. However, as of now, the nuxt-storybook module is not yet compatible with Nuxt 3, and I am not sure wich version it will support, when it is ready - I am planning to update this Project to Storybook 7 (if I find time to do so).

In the meantime, you may need to manually import your Nuxt components into Storybook.

**Comment by alvinzjoe** [comment]
@lazercaveman Exactly! We need to add few script that act like auto import (until nuxt-storybook supported it) on `.storybook/preview.js`:

`const pageContext = import.meta.globEager('../components/**/*.vue');
Object.keys(pageContext).forEach((key) => {
    const paths = key
        .split('/')
        .filter((path) => path !== '..' && path !== 'components');
    const name = paths.pop().split('.')[0];
    return app.component(name, pageContext[key].default);
})`
